pkg/operator/starter.go

```diff
@@ -90,14 +90,6 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 		kubeInformersClusterScoped,
 	)
 
-	configObserver := NewConfigObserver(
```

Comment:
intentional?

Reply:
No, its there and not removed. May be you are looking at specific commit. Just look at the file for final changes.